docs/devel/kubectl-conventions.md

```diff
@@ -135,6 +135,77 @@ Updated: 8/27/2015
 * Use "TYPE" for the particular flavor of resource type accepted by kubectl, rather than "RESOURCE" or "KIND"
 * Use "NAME" for resource names
 
+## Command structure conventions
```

Comment:
Command implementation conventions

You need to run hack/update-generated-docs.sh in order to update the TOC.